controllers/chat/sendSignedMessage.js

```diff
@@ -172,17 +172,21 @@ const sendSignedMesage = async (req, res) => {
       dataPayload.attachment = '';
     }
 
-    BetterSocialCore.fcmToken.sendChatNotificationByChannelMembers(
-      channelMember,
-      {
-        notification: notificationPayload,
-        data: dataPayload
-      },
-      {
-        priority: 'high',
-        content_available: true
-      }
-    );
+    BetterSocialCore.fcmToken
+      .sendChatNotificationByChannelMembers(
+        channelMember,
+        {
+          notification: notificationPayload,
+          data: dataPayload
+        },
+        {
+          priority: 'high',
+          content_available: true
+        }
+      )
+      .catch((e) => {
+        console.log('error send chat notification', e.code);
+      });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

While the addition of error handling is a good practice, using `console.log` for logging errors in production code is not recommended. It's better to use a dedicated logging library that can provide more context and control over log levels, formats, and destinations. Also, it would be beneficial to rethrow the error after logging it so that the caller function can handle it appropriately.

```diff
-       .catch((e) => {
-         console.log('error send chat notification', e.code);
-       });
+       .catch((e) => {
+         logger.error('Error sending chat notification', { errorCode: e.code });
+         throw e;
+       });
```
In this diff, `logger` is an instance of a logging library such as Winston or Bunyan. This change will ensure that the error is logged properly and then rethrown for further handling by the caller.


<!-- This is an auto-generated comment by OSS CodeRabbit -->